extension-style/src/main/java/com/mapbox/maps/extension/style/layers/generated/SkyLayer.kt

```diff
@@ -917,7 +989,6 @@ class SkyLayer(override val layerId: String) : SkyLayerDsl, Layer() {
      *
      * This is an Expression representation of "sky-atmosphere-sun".
      *
-     * Position of the sun center [a azimuthal angle, p polar angle]. The azimuthal angle indicates the position of the sun relative to 0 degree north, where degrees proceed clockwise. The polar angle indicates the height of the sun, where 0 degree is directly above, at zenith, and 90 degree at the horizon. When this property is ommitted, the sun center is directly inherited from the light position.
```

Comment:
why are these sections being removed?

Reply:
Duplicate doc entry.

Reply:
I don't think we should remove these sections, they are duplicated but also makes this API doc self-contained, user don't need to look for another function to see what it does.